src/engraving/libmscore/repeatlist.cpp

```diff
@@ -1027,6 +1031,49 @@ void RepeatList::unwind()
     }
 
     updateTempo();
-    _expanded = true;
+    m_expanded = true;
+}
+
+const Measure* RepeatList::playbackMeasure(size_t staffIdx, const Measure* measure) const
```

Comment:
Not quite undesrtand what's going on here. I mean, the method requires a measure and provides another measure which is considered as "playback" measure.

Could you explain your motivation? :)

Reply:
Suppose you have a score like this: 
<img width="841" alt="Schermafbeelding 2022-03-27 om 15 25 00" src="https://user-images.githubusercontent.com/48658420/160283660-9631d895-d5df-444d-a16f-71bc05bb20c9.png">
For the first two measures, you will just hear those measures themselves. But for the third measure, you should hear the first measure. So, the third measure will appear as a key in the map, with the first measure as its value. 
The same goes for the fourth and second measure: for the fourth measure, you should hear the second measure, so the fourth measure will appear as a key with the second measure as its value. 

And since measure repeats are a per-staff thing, there will be a separate map for each staff.